_source/_posts/2021-01-24-osgeo-glossary-updated.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: custom-post
+title:  "First release of OSGeo Glossary (and how to cite)!"
+date:   2021-01-24 11:30:00 +0800
+categories: update
+---
+
+A new release to celebrate the new year:
+the https://wiki.osgeo.org/wiki/Lexicon_Committee[OSGeo Lexicon Committee]
+has released its first set of terms!
+
+The OSGeo Glossary is now filled with terms and ready to use.
+These terms have been consolidated from OSGeo projects and
+especially the OSGeo Geo4All newsletter.
+
```

Comment:
The OSGeo Glossary comprises new and updated terms with definitions and examples that are all ready to use. 
We have consolidated all these terms from OSGeo projects and especially the OSGeo Geo4All newsletter.